.github/workflows/gotests.yaml

```diff
@@ -36,7 +36,7 @@ jobs:
         run: make git2go
 
       - name: Run tests
-        run: make test
+        run: make test-unit
```

Comment:
Just saw this now, to help with PR check clarity, let's rename `Run rests` to `Run unit tests`:

```diff
- name: Run tests
+ name: Run unit tests
```

Reply:
Did some changes to the wording in d83b3274e105b9c7e77eb7043afa021cf504d9f5:

![image](https://user-images.githubusercontent.com/3299086/235317819-e998b4a0-ce15-49b5-b546-2b3093854e3f.png)
![image](https://user-images.githubusercontent.com/3299086/235317879-fc0c0be8-375d-4a68-8429-e94536d0e0ce.png)

Feel free to fixup and force-push.